assets/js/base/components/cart-checkout/order-summary/order-summary-item.tsx

```diff
@@ -72,13 +77,13 @@ const OrderSummaryItem = ( { cartItem } ) => {
 
 	const regularPriceSingle = Dinero( {
 		amount: parseInt( prices.raw_prices.regular_price, 10 ),
-		precision: parseInt( prices.raw_prices.precision, 10 ),
+		precision: prices.raw_prices.precision,
```

Comment:
Are you certain this will always be an int?

Reply:
Yes, this is typed for a `number`[ here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/js/types/type-defs/cart.ts#L97) so `parseInt()` actually complains that it's receiving a `number` instead of `string`. I thought safe to remove as it should get caught out by type error if anything other than a number is passed to it

Reply:
Sorry I actually meant: are you sure that _at runtime_ it will always be an `number`, TypeSceipt only works at compile time. Do we need a check to cater for the chance that it might be a string or is it fine?

Reply:
You're right, I've changed this now to check for a string :)